wod_predictor/feature_engineering_parts/misc.py

```diff
@@ -0,0 +1,21 @@
+class DropFeatures:
+    def __init__(self, threshold=0.9):
+        self.threshold = threshold
+        self.features_to_drop = []
+
+    def fit(self, df):
+        """
+        identifies features with more than threshold percentage of missing values 
+        """
+        self.features_to_drop = df.columns[df.isnull().mean() > self.threshold]
+
+    def transform(self, df):
```

Comment:
Do you think it's good to add an extra parameter for the method so that you can pass the columns to consider dropping. Other columns won't be dropped even if NaN values are above threshold. This may be useful in the future.

Reply:
Thats a good thought. I'd say there is a small chance it could be useful. Don't want to expose more params than needed in the config to avoid making it unnecessarily complicated. Prefer to keep it as lean as possible.

If this functionality ends up being required later on we can add it.

Reply:
Sound plausible to me. I think I don't have any more comments and this feature should be ready to go.